lib/default-services.js

```diff
@@ -25,35 +25,48 @@ module.exports = function(options, cb) {
   var serverFileLinesCount = serverFileContent.split('\n').length;
   var inclusionString = '';
   var newDependencies = {};
+  var addAutoScaling = false;
+  var addAppMetrics = false;
   if (options.enableAutoScaling) {
-    newDependencies['bluemix-autoscaling-agent'] = '^1.0.7';
-    var autoScalingModuleTemplate = fs.readFileSync(path.resolve(
-      bluemixTemplatesDir, 'services', 'autoscaling-module.tpl'));
-    inclusionString += autoScalingModuleTemplate;
+    if (serverFileContent.indexOf("require('bluemix-autoscaling-agent')") === -1) {
```

Comment:
Assuming this module requires Node.js 4+, I think it's better to use [`String.prototype.includes()`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes) instead of `indexOf()`.